changelog.txt

```diff
@@ -1,7 +1,7 @@
 0.84.2
- * Fixed a bug making it impossible to read Realms created by iOS if a class contained primary keys (#1703).
  * Added Realm.isEmpty().
- * Fixed bug making it impossible to convert a field to being required during a migration.
+ * Fixed a bug making it impossible to convert a field to being required during a migration (#1695).
+ * Fixed a bug making it impossible to read Realms created by iOS if a class contained primary keys (#1703).
```

Comment:
Fixed a bug making it impossible to read Realms using primary keys and created from iOS